lib/ansible/modules/fetch.py

```diff
@@ -76,7 +88,8 @@
   safe_file_operations:
     support: none
   vault:
-    support: none
+    support: full
```

Comment:
I'm not sure what the implications of `support: full` are- @bcoca?

Reply:
It's mostly just documentation, there's no strict standard AFAIK.

Reply:
Stolen from copy module :-), feel free to change to whatever suits better.

Reply:
```    vault:
      description: Can automatically decrypt Ansible vaulted files```
in this case it is 'encrypt'

Reply:
Where is this description coming from? What do you suggest to put to the description instead of full then, "can encrypt into vaulted files" ?

Reply:
https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/action_common_attributes.py#L50

Reply:
now i'm thinking we need `vault:` with choices `decrypt/encrypt/full`

Reply:
and for fetch ... what about a feature that decrypts automatically the fetched content if it is a vault? this is what we normally do in other actions. this is the first time we are looking at encrypting from the action itself (i tried to avoid this by adding the `vault` filrer).

Reply:
OK, I will put there "encrypt" and see what happens :-).

Reply:
Does not pass sanity test without updating the attribute definitions. Should I update these as well? A bit out of scope for this ticket, though.
On the other hand, not sure how important this is, IMHO decryption in fetch does not make sense (as encryption in copy module, also not implemented AFAIK), so specifying full here is probably good as well.